### PR TITLE
templates for wiki_qa dataset

### PR DESCRIPTION
**Pull request by lintangsutawika**

### CODE REVIEW

File: templates/wiki_qa/templates.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      {% endif %}
+
+      '
+    name: Jeopardy style
+    reference: Given a passage (an answer), generate the question.
+  8a9f2146-aa30-4e17-b1e2-aeb858b08b55: !Template
+    id: 8a9f2146-aa30-4e17-b1e2-aeb858b08b55
+    jinja: '{% if label == 1 %}
+
+      Determine if the topic of the question-answer pair.
```
Comment (VictorSanh) [inline templates/wiki_qa/templates.yaml:30]: typo: no need for "if"

File: templates/wiki_qa/templates.yaml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+      Question: "{{question}}?";  Answer: "{{answer}}"?|||
```
Comment (VictorSanh) [inline templates/wiki_qa/templates.yaml:32]: `|||` -> `Topic: ||| "{{document_title}}"

File: templates/wiki_qa/templates.yaml (hunk 1 above)
Comment (VictorSanh) [reply templates/wiki_qa/templates.yaml:30]: same for Topic Prediction - Question only

File: templates/wiki_qa/templates.yaml (hunk 2 above)
Comment (VictorSanh) [reply templates/wiki_qa/templates.yaml:32]: same for Topic Prediction - Question only

**Review by VictorSanh — CHANGES_REQUESTED** [review]
Thanks @lintangsutawika, lgtm!

Flagged some small typos in comments, I'll merge once you have changed these

**Comment by lintangsutawika** [comment]
@VictorSanh Sorry for the typos. They are fixed now.

**Review by VictorSanh — APPROVED** [review]

**Comment by VictorSanh** [comment]
thanks, merging now!

**Pull request merged**